### ISSUE TITLE
A11y_mssql for VSCode_Results Tab_Screen Reader: Jaws is not reading row numbers and column numbers while navigating with in the grid

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/vscode-mssql#1233]
GitHubTags: #A11y_mssqlforVSCode; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #Jaws; #ScreenReader; #MAS1.3.1; #mssqlforVSCode

**Environment:**
Visual Studio Code – Insider (install from https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2Finsiders.&data=02%7C01%7C%7C00d9f7de52714b323f9908d607afe0c5%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636704849297354687&sdata=cmMqckjoNPGtCS3SZ00bCqCnFprKgiux%2BIjyvzgU0hE%3D&reserved=0)
Version: 1.34.0-insider (user Setup)
Mssql version: 1.6.0
OS Version: Microsoft Windows 10 Enterprise (10.0.17763 Build 17763)
Screen Reader: Jaws

**MAS ID Violated:** MAS 1.3.1

**Repro Steps:**
1. Open Visual Studio Code in your system
2. Navigate to "Extensions" and install "SQl Server (mssql)" extension
3. Press F1 and select MS SqL : Connect and connect to any sql server
4. "Write  code (SELECT *
FROM sys.objects;) in the new document and press F1 and select ""MS SQL : Execute Query""  "
5. While navigating within the grid verify Jaws is reading the row numbers & column numbers  or not

**Actual Result:**
Jaws is not reading row numbers and column numbers while navigating in the grid

**Expected Result:**
Jaws should read row numbers and column numbers while navigating in the grid

**User Impact:**
Screen Reader users finds difficult in understanding about the table if Jaws does not read row numbers & column numbers

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachments:**
![Bug_1233_jaws not reading row and column numbers](https://user-images.githubusercontent.com/42638450/56648737-775f1a80-66a1-11e9-8bfe-551ff7f50306.png)

**Comment by kenvanhyning** [comment]
This should be fixed in 1.7 though the indexing read is 0 based rather than 1 based.  Please advise if this is not acceptable.

**Comment by BhanuPrakashGit** [comment]
GithubTag:#A11ySev2;

**Comment by Roopesh-T** [comment]
issue is fixed in 
Version: 1.79.0-insider (user setup)
so closing the bug
**GitTag**: #closed;
[1233-Closed.webm](https://github.com/microsoft/vscode-mssql/assets/86601127/038a0052-4650-4c5f-be30-8b10029e78c4)
